tests/rt.sh

```diff
@@ -381,7 +381,7 @@ elif [[ $MACHINE_ID = orion ]]; then
 
 elif [[ $MACHINE_ID = hercules ]]; then
 
-  module load contrib rocoto/1.3.5
+  module load contrib rocoto
```

Comment:
@MatthewPyle-NOAA This is what I'm talking about. Does this need to go to the develop branch?